openreview/venue_request/venue_request.py

```diff
@@ -164,14 +164,55 @@ def setup_review_stage(self):
                 'default': 'No, do not email program chairs about received reviews',
                 'order': 27
             },
+            'review_rating_field_name': {
+                'description': "Name of the rating field for the review invitation, default is \'Rating\'",
```

Comment:
You can add the description here: "Please follow the format number: description if you want to customize options and add one option per line."